cmd/utils/manifest.go

```diff
@@ -31,3 +31,12 @@ func InferName(cwd string) string {
 	oktetoLog.Info("inferring name from git repository URL")
 	return model.TranslateURLToName(repo)
 }
+
+// GetWorkdirFromManifestPath sets the path
+func GetWorkdirFromManifestPath(manifestPath string) string {
+	dir := filepath.Dir(manifestPath)
+	if filepath.Base(dir) == ".okteto" {
```

Comment:
I could do `okteto deploy -f ~/app/.okteto/okteto.yml`, should we cover that case?

Reply:
it will be executed from `~/app/`. Do we want to run it from `~`?

Reply:
It should be `~/app/`, perfect!

Reply:
I will add a test for that before merging